frontend/src/components/Alerts/InlineAlert.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+export enum alertType {
+  CUSTOM = 'custom',
+  INFO = 'info',
+  SUCCESS = 'success',
+  WARNING = 'warning',
+  DANGER = 'danger'
+}
+
+type InlineAlertProps = {
+  message: string;
+  type: alertType;
+};
+
+export const InlineAlert: React.FC<InlineAlertProps> = (props: InlineAlertProps) => {
```

Comment:
Why did you decide to use this impl instead of the standard PF inline alert? 

https://www.patternfly.org/components/alert#inline-alerts-variants

Reply:
That was to use just the pf5 ones, right? https://v5-archive.patternfly.org/components/alert/html
(The one in your link is pf6)